lib/plaid/connection.rb

```diff
@@ -9,22 +9,29 @@ class << self
       def post(path, options = {})
         uri = build_uri(path)
         options.merge!(client_id: Plaid.customer_id, secret: Plaid.secret)
-        res = Net::HTTP.post_form(uri,options)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = (uri.scheme == 'https')
```

Comment:
A quote from Plaid doc: "The Plaid API is served over HTTPS to ensure data privacy; HTTP is not supported. "
I think this http vs https logic could be safely replaced for strict http.